### DISCUSSION TITLE
[Feature] Select folder to upload files

### DISCUSSION THREAD

**Discussion by fedyk** [immich-app/immich#7015] [Feature Request]
### The feature

Currently, web app allows only to upload files. [webkitdirectory](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitdirectory) allows to select a folder and have a list of files. 

With `webkitdirectory`, use can only select folders. To preserve the option to upload specific files, two options should be available on UI. Perhaps, some drop menu with options to select between files and folders.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
I didn't know this, very good. I like the idea of having different options to drag and drop when it is a folder. So we can either upload individual images in the folder or create the album with the folder name

**Comment by bo0tzz** [comment]
Duplicate of #5938
